### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by E11NTM** [MattFiler/OpenCAGE#374]
So I'm trying to change items on cathode scripting for example a noise maker but the thing is that only works on one level, so for example if I modify the Noise Maker on one level it will only work on that one which makes sense. But then I would have to change the noise maker manually on each level. It would be nice for a feature of changing globally a file that theres a bunch of levels that share in common that way you save a lot of time.

**Comment by MattFiler** [comment]
So OpenCAGE did used to load all levels in the asset editor so that changes to textures/models were applied to all, but it becomes tricky to manage.

Imagine someone downloaded a custom map someone else had made - not every level then would likely have the same content across the board. I'd have to add ways to detect varying assets with the same names across levels.

It becomes even more complicated when it comes to scripting, as the number of changes are potentially huge and apply across multiple composites.

I'll leave this ticket open though as it seems like a good feature to have at some point, even if it's just covered by the improved porting functionality (#247).

**Comment by E11NTM** [comment]
Yeah! Maybe making multiple files being able to be ported to multiple levels would definitely work for me. As long as I can port to other levels the files that I edited in one level and I want it on the rest works.
